napalm_brocade/brocade.py

```diff
@@ -0,0 +1,428 @@
+#
+# Copyright 2016 Shiv Haris, Brocade Communication Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License.  You may obtain a copy of
+# the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+"""
+Brocade-Napalm Driver.
+
+This driver is meant for SLX and NOS based switches.
+"""
+from netmiko import ConnectHandler
+from napalm_base.base import NetworkDriver
+from napalm_base.exceptions import ConnectionException, MergeConfigException, \
+    ReplaceConfigException, SessionLockedException, CommandErrorException
+
+import os
+import re
+from shutil import copyfile
+
+
+# TBD(shh) Put this in config file (oslo_config)
```

Comment:
Definitely remove this - don't need to have any hardcoded info, not even in a config file.